pontoon/machinery/views.py

```diff
@@ -59,11 +59,9 @@ def translation_memory(request):
     locale = get_object_or_404(Locale, code=locale)
     entries = (
         TranslationMemoryEntry.objects
-        .minimum_levenshtein_ratio(text)
-        .filter(locale=locale)
+        .minimum_levenshtein_ratio(locale, text)
```

Comment:
If you were to move `filter(locale)` before the call to `minimum_levenshtein_ratio`, wouldn't that allow you to simply ignore the `locale` filter altogether in the levenshtein functions? Because you'd trust that whatever `self` has is already filtered for that locale.